source/Calamari/Integration/EmbeddedResources/CallingAssemblyEmbeddedResources.cs

```diff
@@ -4,16 +4,16 @@
 
 namespace Calamari.Integration.EmbeddedResources
 {
-    public class ExecutingAssemblyEmbeddedResources : ICalamariEmbeddedResources
+    public class CallingAssemblyEmbeddedResources : ICalamariEmbeddedResources
```

Comment:
Because Calamari is no longer a single assembly, embedded-resources now need to be able to be loaded from an assembly that isn't necessary the executing-assembly.  Calling-assembly works for now.